ingest/downloader/downloader.py

```diff
@@ -43,10 +43,8 @@ def add_worksheet_content(self, worksheet, ws_elements: dict):
 
     def __add_header_row(self, worksheet, headers: list):
         self.row = HEADER_ROW_NO
-        col = 1
-        for header in headers:
-            worksheet.cell(row=self.row, column=col, value=header)
-            col += 1
+        for col, header in enumerate(headers):
```

Comment:
`enumerate()` is zero based by default.

```suggestion
        for col, header in enumerate(headers, start=1):
```

Reply:
This suggestion would break the code